rfcs/0000-safe-transmute.md

```diff
@@ -715,18 +715,18 @@ The default value of the `Neglect` parameter, `()`, statically forbids transmute
 | `NeglectAlignment`  | Safety      | `unsafe_transmute_{from,into}`                          |
 | `NeglectValidity`   | Soundness   | `unsafe_transmute_{from,into}`                          |
 
-`NeglectStabilty` implements the `SafeTransmuteOptions` and `UnsafeTransmuteOptions` marker traits, as it can be used in both safe and unsafe code. The selection of multiple options is encoded by grouping them as a tuple; e.g., `(NeglectAlignment, NeglectValidity)` is a selection of both the `NeglectAlignment` and `NeglectValidity` options.
+`NeglectStabilty` implements the `SafeTransmuteOptions` and `TransmuteOptions` marker traits, as it can be used in both safe and unsafe code. The selection of multiple options is encoded by grouping them as a tuple; e.g., `(NeglectAlignment, NeglectValidity)` is a selection of both the `NeglectAlignment` and `NeglectValidity` options.
```

Comment:
FWIW: I think a more common naming scheme in Rust is for the default to be safe and non-default to be explicitly marked as unsafe. Would it be better for `TransmuteOptions` to be safe and for there to be `UnsafeTransmuteOptions`